### DISCUSSION TITLE
[Feature] Access logging and accessible data for sharing links

### DISCUSSION THREAD

**Discussion by sym0nd0** [immich-app/immich#10920] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have recently shared some links to albums on my instance with family and friends. It would be nice to be able to see how many times in total each individual sharing link has been accessed, capture the IPs of each access (for blocking if required) and show for Admin and album/sharing link owner how many people are actively using that link to view the album at that time.

Apologies if this is a duplicate, I've searched as best I can but nothing seems to be similar.

Thanks.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10055

**Reply by sym0nd0** [reply]
I saw that one, but honestly couldn't fully understand what the poster was asking for and it didn't include the detail of my request.
